From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: fix prefix

arora installs to /usr/local by default.
This patch changes this to /usr.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
# 20091210 rsc: is this patch ready for upstream, or should it be solved
#          differently?

 install.pri |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/install.pri
===================================================================
--- a/install.pri
+++ b/install.pri
@@ -1,7 +1,7 @@
 
 unix {
     isEmpty(PREFIX) {
-        PREFIX = /usr/local
+        PREFIX = /usr
     }
     BINDIR = $$PREFIX/bin
 
